### PR TITLE
Kiet huynh

### PR DESCRIPTION
**Pull request by KHONGPHAITKID**

### CODE REVIEW

File: include/CountingSort.h
Comment (winprn) [inline include/CountingSort.h:1]: cstring for what?

File: include/SortContext.h
Comment (winprn) [inline include/SortContext.h:1]: remove startTimer, stopTimer. Assign m_timeStart  = m_timeEnd = 0 in sorting functions instead

File: include/SortStrategy.h
Comment (winprn) [inline include/SortStrategy.h:1]: close the stream after finished

File: src/CommandLine.cpp
Comment (winprn) [inline src/CommandLine.cpp:1]: remove debugging comments

File: include/SortContext.h
Comment (KHONGPHAITKID) [reply include/SortContext.h:1]: done sir!

File: include/SortStrategy.h
Comment (KHONGPHAITKID) [reply include/SortStrategy.h:1]: ok sir!

File: src/CommandLine.cpp
Comment (KHONGPHAITKID) [reply src/CommandLine.cpp:1]: i have done it !

**Pull request merged**